src/lang/locales/en-US.json

```diff
@@ -172,7 +172,9 @@
   "parameters": {
     "default": "Default",
     "minimum": "Minimum",
+    "minimumLength": "Minimum Length",
     "maximum": "Maximum",
+    "maximumLength": "Maximum Length",
     "possible-types": "Type | Possible types",
     "possible-values": "Value | Possible Values"
```

Comment:
Not strictly related to this PR, but the casing is inconsistent across these parameter labels, e.g., Possible **t**ypes vs. Possible **V**alues. Which casing should we adopt?

Reply:
I would be partial to `"Minimum length"` myself, but I was trying to match how I had seen it before in another place. These are default strings for the scenario where DocC doesn't provide the titles itself in the `title` field, which should be the more common scenario.